TemporaryEmail/source/info.plist

```diff
@@ -90,7 +90,9 @@
 				<integer>102</integer>
 				<key>script</key>
 				<string>function notification {
-  ./notificator --message "${1}" --title "${alfred_workflow_name}"
+  if [[ "${notifications_enabled}" == 'true' ]]; then
```

Comment:
The notification is pretty annoying for me and useless. Also macOS asks me to enable notifications for `Notificator` which is annoying.

Reply:
If you don’t want notifications, refuse them when macOS asks. There’s no reason to add more code and configuration for something macOS deals with.

Reply:
Hmm good point, I remember it messed up another workflow but I'm not entirely sure why anymore… I'll try and report back.